CRM/Core/Payment.php

```diff
@@ -1277,9 +1277,24 @@ public function doPayment(&$params, $component = 'contribute') {
    *
    * @param array $params
    *
+   * @return array
    * @throws \Civi\Payment\Exception\PaymentProcessorException
    */
-  public function doRefund(&$params) {}
+  public function doRefund($params) {
```

Comment:
This might also need a change in [CRM/Core/Payment/Dummy.php](https://github.com/civicrm/civicrm-core/blob/master/CRM/Core/Payment/Dummy.php#L171) to have a compatible declaration?

Reply:
@jitendrapurohit Thanks I've added that in